src/scheduler/main.py

```
@@ -435,552 +445,578 @@ def run_in_slave_mode():
+
+                            SCHEDULER.send_to_apis("POST", "/reload")
+                    else:
+                        logger.info("Successfully reloaded bunkerweb")
+                        # Update the failover path with the working configuration
+                        rmtree(FAILOVER_PATH, ignore_errors=True)
+                        FAILOVER_PATH.mkdir(parents=True, exist_ok=True)
+                        copytree(CONFIG_PATH, FAILOVER_PATH.joinpath("config"))
+                        copytree(CUSTOM_CONFIGS_PATH, FAILOVER_PATH.joinpath("custom_configs"))
+                        copytree(CACHE_PATH, FAILOVER_PATH.joinpath("cache"))
+                        Thread(target=JOB.cache_dir, args=(FAILOVER_PATH,), kwargs={"job_name": "failover-backup"}).start()
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)